### ISSUE TITLE
GUI 1230 FrTX 1.1 - 3-axis gyro and 3-axis accelerator

### ISSUE THREAD

**Issue by MIKE-DAILEY** [FrSky-OS/discussion#58]
Do not see controls for the 3-axis gyro and 3-axis accelerator

**Comment by MIKE-DAILEY** [comment]
In the “Input Map” screen there is a selection for input “G_Roll”, “G_Pitch”.  Is it possible that they are the input from the gyro?  It also looks like “G_Roll” and "G_Pitch" is on the list twice.  Can’t find anything that looks like an accelerator input.

**Comment by janekx** [comment]
Yes cant see any accelerometer or gyro screen, would be nice to see some position and calibration of these sensors. Like overview ....
